src/source/agc.rs

```diff
@@ -183,6 +183,66 @@ where
     I: Source,
     I::Item: Sample,
 {
+    #[inline]
+    fn target_level(&self) -> f32 {
+        #[cfg(feature = "experimental")]
+        {
+            self.target_level.load(Ordering::Relaxed)
+        }
+        #[cfg(not(feature = "experimental"))]
+        {
```

Comment:
I do not think these `{}` are necessary, does it work without?

Reply:
It does not work without it, I get errors with it wanting me to append ; to the end of the experimental one so for consistency I made them both the same easier to read in the future since one is needed anyway

Reply:
might be a rust analyzer issue. W'll merge and then ill play around with it a little :)